<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffmpeg_testClass</class>
 <widget class="QMainWindow" name="ffmpeg_testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ffmpeg_test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>IPC相机</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>大恒相机</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4K相机</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>相机URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>rtsp://admin:hk123456@192.168.0.11</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输出路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>G:\\test.avi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QLabel" name="label_image">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>600</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>结束录像</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>76</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开始录像</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>打开相机</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>124</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>结束显示</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>播放视频</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ffmpeg_test.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ffmpeg_testClass</receiver>
   <slot>start_record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ffmpeg_testClass</receiver>
   <slot>close_record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ffmpeg_testClass</receiver>
   <slot>slot_open_camera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ffmpeg_testClass</receiver>
   <slot>slot_close_camera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>ffmpeg_testClass</receiver>
   <slot>vedio_player()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>786</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_record()</slot>
  <slot>close_record()</slot>
  <slot>slot_open_camera()</slot>
  <slot>slot_close_camera()</slot>
  <slot>vedio_player()</slot>
 </slots>
</ui>
